apps/homepage/src/app/[lang]/contact/contactForm.tsx

```diff
@@ -75,14 +81,16 @@ export function ContactForm() {
         )}
       </div>
       <div className={styles.inputContainer}>
-        <CustomTextArea
+        <XDevTextArea
           name="message"
           id="message-area"
           label={loc.messageInput}
           icon="Send"
           value={contactInfo.message}
           onChange={handleChangeInputs}
+          maxLength={charLimit}
         />
+        <span className={styles.characterCount}>{loc.maxLengthMessage} {charCount}</span>
```

Comment:
The counter should be part of the text area and you can use the prop `maxLength` to get the value.